.github/workflows/scheduled-aws-experiment-pipeline.yml

```
@@ -0,0 +1,243 @@
+        with:
+          go-version: '1.14'
+
+      - name: Create Kubernetes secret for aws experiment
+        if: always()
+        env:
+          KUBECONFIG: /etc/rancher/k3s/k3s.yaml      
+        run: |
+          cat <<EOF | kubectl apply -f -
+          apiVersion: v1
+          kind: Secret
+          metadata:
+            name: cloud-secret
+            namespace: default
+          type: Opaque
+          stringData:
+            cloud_config.yml: |-
+              [default]
+              aws_access_key_id = ${{ secrets.AWS_ACCESS_KEY_ID }}
+              aws_secret_access_key = ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          EOF
+          
+      - name: Configure AWS Credentials
+        if: always()
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          # aws-session-token: ${{ secrets.AWS_SESSION_TOKEN }} # if you have/need it
+          aws-region: ${{ secrets.REGION }}
+          
+      - name: Creating target EC2 instances
+        if: always()
+        run: |
+          aws ec2 run-instances \
+          --image-id ami-0d382e80be7ffdae5 \
+          --count 2 --instance-type t2.nano \
+          --key-name udit-litmus-e2e \
+          --security-group-ids sg-06925d6218d07e795 \
+          --subnet-id subnet-9c2ba8fa \
+          --region ${{ secrets.REGION }} \
+          --tag-specifications 'ResourceType=instance,Tags=[{Key=runNumber,Value=${{ github.run_number }}}]'
+          
+      - name: Get the target instance name
+        if: always()
+        run: |
+          instances=$(aws ec2 describe-instances --filters Name=tag-value,Values=${{ github.run_number }} Name=instance-state-name,Values=pending,running --region ${{ secrets.REGION }} --query 'Reservations[*].Instances[*].{Instance:InstanceId}' --output text)
+          instance_list=($instances)
+          echo "inst1: ${instance_list[0]} inst2: ${instance_list[1]}"
+          echo "INSTANCE_ONE=${instance_list[0]}" >> $GITHUB_ENV
+          echo "INSTANCE_TWO=${instance_list[1]}" >> $GITHUB_ENV
+          
+      - name: Litmus Infra Setup
+        if: always()
+        run: make build-litmus
+        env:
+          KUBECONFIG: /etc/rancher/k3s/k3s.yaml
+
+      - name: Run EC2 Terminate By ID experiment in serial & parallel mode
+        if: always()
+        env:
+          KUBECONFIG: /etc/rancher/k3s/k3s.yaml
+          EC2_INSTANCE_ID: "${{ env.INSTANCE_ONE }},${{ env.INSTANCE_TWO }}"
+          REGION: ${{ secrets.REGION }}
+        run: make ec2-terminate-by-id
+
+      - name: Run EC2 Terminate By Tag experiment in serial & parallel mode
+        if: always()
+        env:
+          KUBECONFIG: /etc/rancher/k3s/k3s.yaml
+          EC2_INSTANCE_TAG: "${{ env.INSTANCE_TAG }}"
+          REGION: ${{ secrets.REGION }}
+        run: make ec2-terminate-by-id       
```

This should be by Tag
```suggestion
        run: make ec2-terminate-by-tag      
```